src/%ZPM/PackageManager/Developer/Processor/WSGIApplication.cls

```diff
@@ -0,0 +1,46 @@
+Include (%sySecurity, %occErrors)
+
+Class %ZPM.PackageManager.Developer.Processor.WSGIApplication Extends %ZPM.PackageManager.Developer.Processor.CSPApplication
+{
+
+Parameter ATTRIBUTES = {""_
```

Comment:
@isc-shuliu I would recommend changing this to be generic WebApplication rather than something specific for WSGI. Also, I created an initial version of such a generic one here: 
//projects/healthshare/appmodules/HSIEO-WIP/ZHSLIB/databases/hscommon/cls/ZHSLIB/PackageManager/Developer/Processor/WebApplication.cls

The approach in the file above, lets the processor class automatically include and support any new props of Security.Applications instead of a hardcoded list. Furthermore, it doesn't do automatic copying of files (which I think is good). Since the user can use the FileCopy resource processor if they want that. This limits side effects and allows them to just create a web app.

Apologies for the delay with the review. Wish I had gotten to it before you checked it in.

cc @isc-tleavitt